src/Text/LLVM/PP.hs

```diff
@@ -74,6 +74,10 @@ checkConfig :: (?config :: Config) => (Config -> Bool) -> Bool
 checkConfig p = p ?config
 
 
+-- | This type encapsulates the ability to convert an object into Doc
+-- format. Using this abstraction allows for a consolidated representation of the
+-- declaration.  Most pretty-printing for LLVM elements will have a `Fmt a`
```

Comment:
Haddock nitpick: backticks behave like single quotes in that they hyperlink particular identifiers. If you want just to render this in a monospace font, I think you want:

```hs
-- declaration.  Most pretty-printing for LLVM elements will have a @Fmt a@
```

Or, if you want `Fmt` to be hyperlinked:

```hs
-- declaration.  Most pretty-printing for LLVM elements will have a @'Fmt' a@
```